src/organizercore.cpp

```diff
@@ -892,66 +909,22 @@ QStringList OrganizerCore::modsSortedByProfilePriority() const
 
 void OrganizerCore::spawnBinary(const QFileInfo &binary, const QString &arguments, const QDir &currentDirectory, bool closeAfterStart, const QString &steamAppID)
 {
-  LockedDialog *dialog = new LockedDialog(qApp->activeWindow());
-  dialog->show();
-  ON_BLOCK_EXIT([&] () { dialog->hide(); dialog->deleteLater(); });
+  if (m_UserInterface != nullptr) {
+    m_UserInterface->lock();
+  }
+  ON_BLOCK_EXIT([&] () {
+    if (m_UserInterface != nullptr) { m_UserInterface->unlock(); }
+  });
 
   HANDLE processHandle = spawnBinaryDirect(binary, arguments, m_CurrentProfile->name(), currentDirectory, steamAppID);
   if (processHandle != INVALID_HANDLE_VALUE) {
-    if (closeAfterStart && (m_UserInterface != nullptr)) {
+    if (closeAfterStart && m_UserInterface != nullptr) {

```

Comment:
Please respect the existing coding style and please don't make unnecessary changes in code that I will have to merge into the branch.
In general, it would have been easier if this diff had been made to the new_vfs_library branch because there has been some changes to the process spawning code.